pkg/plugin/test-keeper/metrics.go

```diff
@@ -0,0 +1,72 @@
+package testkeeper
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	gogh "github.com/google/go-github/github"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/arquillian/ike-prow-plugins/pkg/server"
+)
+
+const (
+	// WithTests is a label used in prometheus metrics pull_requests_total for pull request containing some tests.
+	WithTests = "with_tests"
+	// WithoutTests is a label used in prometheus metrics pull_requests_total for pull request without any tests.
+	WithoutTests = "without_tests"
+)
+
+var (
+	pullRequestsCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pull_requests_total",
+		Help: "Total number of pull requests.",
+	}, []string{"full_name", "type"})
+	okWithoutTestsPullRequest = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "pull_requests_changed_file_size",
+		Help:    "Histogram for changed file size of pull request with ok-without-tests.",
+		Buckets: prometheus.ExponentialBuckets(1, 5, 6),
+	}, []string{"full_name"})
+)
+
+// RegisterMetrics registers prometheus collectors to collect metrics for test-keeper.
+func RegisterMetrics() []error {
+	errors := make([]error, 0, 2)
+	server.RegisterOrGet(pullRequestsCounter, &errors, func(collector prometheus.Collector) {
+		pullRequestsCounter = collector.(*prometheus.CounterVec)
+	})
+
+	server.RegisterOrGet(okWithoutTestsPullRequest, &errors, func(collector prometheus.Collector) {
+		okWithoutTestsPullRequest = collector.(*prometheus.HistogramVec)
+	})
+
+	return errors
+}
+
+func reportPullRequest(l log.Logger, pr *gogh.PullRequest, prType string) {
+	fullName := *pr.Head.Repo.FullName
```

Comment:
This is the label that you use for collecting the counter? Correct me if I'm wrong - this is the repository name the PR was sent from. Could you please explain to me what you are trying to achieve here? From the code, I have a feeling that you count every PR event that is received in `test-keeper` plugin for repository the PR has been sent from separately.

Reply:
similarly here https://github.com/arquillian/ike-prow-plugins/pull/187/files#diff-241f68d9d4673ceec43fbd014ca925acR53